specs/builder.md

```diff
@@ -121,6 +121,45 @@ class SignedBlindedBeaconBlock(Container):
     signature: BLSSignature
 ```
 
+#### Capella
+
+##### `BlindedBeaconBlockBody`
+
+```python
+class BlindedBeaconBlockBody(Container):
+    randao_reveal: BLSSignature
+    eth1_data: Eth1Data
+    graffiti: Bytes32
+    proposer_slashings: List[ProposerSlashing, MAX_PROPOSER_SLASHINGS]
+    attester_slashings: List[AttesterSlashing, MAX_ATTESTER_SLASHINGS]
+    attestations: List[Attestation, MAX_ATTESTATIONS]
+    deposits: List[Deposit, MAX_DEPOSITS]
+    voluntary_exits: List[SignedVoluntaryExit, MAX_VOLUNTARY_EXITS]
+    sync_aggregate: SyncAggregate
+    execution_payload_header: ExecutionPayloadHeader
+    bls_to_execution_changes: List[SignedBLSToExecutionChange, MAX_BLS_TO_EXECUTION_CHANGES]  # [New in Capella]
+```
+
+#### EIP-4844
```

Comment:
I guess we'd also need `blob_kzg_commitments` , perhaps in `BuilderBid`? I'm not sure :/

Reply:
I've added a suggestion below - let's try to get this draft PR over to the main repo so we can get some feedback asap